forge-core/src/main/java/forge/deck/Deck.java

```diff
@@ -94,7 +95,9 @@ public Deck(final Deck other, final String newName) {
         for (final Entry<DeckSection, CardPool> sections : other.parts.entrySet()) {
             parts.put(sections.getKey(), new CardPool(sections.getValue()));
         }
+        aiHints.addAll(other.getAiHints());
```

Comment:
looking at this code, it all seems redundant since `cloneFieldsTo` already does it?
(well only the line for tags doesn't seem duplicated)

Reply:
Ah yep. Good call out.

Reply:
this also applies to the for-loop?